### ISSUE TITLE
Error in warning activity diagram.

### ISSUE THREAD

**Issue by cedricongjh** [cedricongjh/pe#16]
![image.png](https://raw.githubusercontent.com/cedricongjh/pe/main/files/e3fa7a74-4e7c-401f-baba-b301fd419821.png)

There are 2 issues: firstly, the true and false in confirm to proceed with command should have guard clauses.
Secondly, the condition, confirm to proceed with command should not be inside the diamond, rather, it should be outside, as [User confirms to proceed with command] and [else].

To sum up, the guard conditions should be replaced accordingly:

true with [User confirms to proceed with command]
false with  [else]
